dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,45 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    @Test
+    public void transformsTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = null;
+        try {
+            outBigDec = (BigDecimal) bdt.transform(nbd);
+        } catch (SQLException e) {
+            fail("InformixBigDecimalTransformer threw SQLException transforming NormalisedBigDecimal input");
+        }
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsInputTest() {
```

Comment:
There are some [JUnit features for testing thrown exceptions](https://github.com/junit-team/junit/wiki/Exception-testing): to make such tests simpler